can-component.js

```diff
@@ -307,18 +300,39 @@ var Component = Construct.extend(
 							nodeLists.replace([el], subtemplate(lightTemplateData.scope, lightTemplateData.options));
 						}
 
-						// Restore the content tag so it could potentially be used again (as in lists)
-						options.tags.content = contentHookup;
+						// Restore the proper tag function so it could potentially be used again (as in lists)
+						options.tags[tagName] = hookupFunction;
 					}
-				};
+				}
+			}
+
+			// If this component has a view (that we've already converted to a renderer)
+			if (this.constructor.renderer) {
+				// If `options.tags` doesn't exist set it to an empty object.
+				if (!options.tags) {
+					options.tags = {};
+				}
+
+				if (!isEmptyObject(templates)) {
```

Comment:
hmmm, eventually, we probably should support dynamic templates.   No need for now.